daprdocs/content/en/reference/components-reference/supported-state-stores/setup-oracledatabase.md

```diff
@@ -0,0 +1,165 @@
+
+The following operation therefore: 
+
+```shell
+curl -X POST http://localhost:3500/v1.0/state \
+  -H "Content-Type: application/json"
+  -d '[
+        {
+          "key": "temporary",
+          "value": "ephemeral",
+          "metadata": {"ttlInSeconds": "120"}}
+        }
+      ]'
+```
+
+creates the following object:
+
+| KEY | VALUE | CREATION_TIME  |EXPIRATION_TIME  | BINARY_YN | ETAG |
+| ------------ | ------- | ----- | ----- | ---- |---- |
+| temporary | ephemeral | 2022-03-31T22:11:00 |2022-03-31T22:13:00 | N | 79dfb504-5b27-43f6-950f-d55d5ae0894f |
+
+with the EXPIRATION_TIME set to a timestamp 2 minutes (120 seconds) (later than the CREATION_TIME) 
+
+Note that expired state is not removed from the state store by this component. An application operator may decide to run a periodic job that does a form of garbage collection in order to explicitly remove all state records with an EXPIRATION_TIME in the past.
+
+## Concurrency
+
+Concurrency in the Oracle Database state store is achieved by using `ETag`s. Each piece of state recorded in the Oracle Database State Store is assigned a unique ETag - a generated, unique string stored in the column ETag - when it is created or updated (aka replaced). Note: the column UPDATE_TIME is also updated whenever a Set operation is performed on an existing record. When the `Set` and `Delete` requests for this state store specify the FirstWrite concurrency policy, then the request need to provide the actual ETag value for the state to be written or removed for the request to be successful. 
+
+## Consistency
+
+Oracle Database state store supports Transactions. Multiple Set and Delete commands can be combined a single request that is processed as a single, atomic transaction. Note: simple Set and Delete operations are a transaction on their own; when a Set or Delete requests returns an HTTP-20X result, the database transaction has been committed successfully. 
+
+## Query
+
+Oracle Database state store does not currently support the Query API.
+
+
+
+## Create Oracle Database and User Schema
+
+{{< tabs "Self-Hosted" "Autonomous Database on OCI">}}
+
+{{% codetab %}}
+
+1. Run an instance of Oracle Database. You can run a local instance of Oracle Database in Docker CE with the following command:
+
+     This example does not describe a production configuration because it sets the password for users `SYS` and `SYSTEM` in plain text.
+
+     ```bash
+     docker run -d -p 1521:1521 -e ORACLE_PASSWORD=TheSuperSecret1509! gvenzl/oracle-xe
+     ```
+
+2. Create a schema for state data.
+Create a new user schema for storing state data. Grant this user (schema) privileges for creating a table and storing data in the associated tablespace.
```

Comment:
Is this correct "Create a new user schema". Should this be "Create a new user dapr_state". In effect what you say here does not match the SQL below.

Reply:
the term _user schema_ is a common one in the world of Oracle Database. I will make it more explicit by providing an example for the name of the _user schema_.